content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,155 @@
+For high throughput services, in the case of an incident that would require your attention, the issue will show symptoms in multiple traces. As a result, there’s usually no need for you to collect every single trace of a service / endpoint. 
+In order to optimize visibility into trace data and cut down the noise, Datadog [Ingestion Control Mechanisms][1] helps you keep the visibility that you need to troubleshoot problems while remaining within budget.
+
+This guide helps you understand when and how to use ingestion control configurations depending on the main use cases you might be encountering. 
+
+These Ingestion Mechanisms are controllable from the Datadog Agent and from Datadog tracing libraries. If you are using OpenTelemetry SDKs to instrument your applications, you can refer to the dedicated guide on [Ingestion Sampling with OpenTelemetry][2].
+
+## Knowing which ingestion mechanisms are used
+
+In order to identify what current ingestion mechanisms are used in your Datadog environment, navigate to  the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes **by service:**
+
+The configuration column indicating `CONFIGURED` or `AUTOMATIC` provides a first indication of the current set up : 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied for the traces starting from the service. Read more about the specifics of the Datadog Agent logic in the documentation.
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied for the traces starting from the service.
+
+Clicking on the various services to view  details as to what are the sampling decision makers in each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/service_ingestion_summary.png" alt="Service Ingestion Summary" style="width:90%;" >}}
+
+## Configuring ingestion sampling to keep visibility over relevant traces and spans
+
+### Keeping entire transaction traces
+
+#### Why ? 
+
+Ingesting entire transaction traces ensures to keep visibility over the **end-to-end service request flow** for specific individual requests.
+
+#### What solutions does Datadog provide ?
+
+Complete traces can be ingested with [head-based sampling][4] mechanisms: the decision of keeping or dropping the trace is taken at the trace creation (trace head), and that this decision is propagated in the request context to downstream callee services.
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/head_based_sampling_keep.png" alt="Head-based Sampling" style="width:100%;" >}}
+
+
+To decide which requests to keep and drop, the Datadog Agent computes automatic sampling rates for each service to apply at the root of the trace.
```

Comment:
This sentence is hard to understand. Do you want to say that the automatic sampling rate set by the agent is the default behaviour but can be over-ridden if necessary?

Reply:
changed the wording